src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorPicker.xaml

```diff
@@ -0,0 +1,91 @@
+          <DropDownButton.Flyout>
+            <Flyout FlyoutPresenterClasses="NoPadding">
+              <ColorView x:Name="FlyoutColorView"
+                         Color="{Binding Color, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                         ColorModel="{Binding ColorModel, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                         ColorSpectrumComponents="{TemplateBinding ColorSpectrumComponents}"
+                         ColorSpectrumShape="{TemplateBinding ColorSpectrumShape}"
+                         HsvColor="{Binding HsvColor, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                         IsAlphaEnabled="{TemplateBinding IsAlphaEnabled}"
+                         IsAlphaVisible="{TemplateBinding IsAlphaVisible}"
+                         IsColorComponentsVisible="{TemplateBinding IsColorComponentsVisible}"
+                         IsColorModelVisible="{TemplateBinding IsColorModelVisible}"
+                         IsColorPaletteVisible="{TemplateBinding IsColorPaletteVisible}"
+                         IsColorPreviewVisible="{TemplateBinding IsColorPreviewVisible}"
+                         IsColorSpectrumVisible="{TemplateBinding IsColorSpectrumVisible}"
+                         IsColorSpectrumSliderVisible="{TemplateBinding IsColorSpectrumSliderVisible}"
+                         IsComponentSliderVisible="{TemplateBinding IsComponentSliderVisible}"
+                         IsComponentTextInputVisible="{TemplateBinding IsComponentTextInputVisible}"
+                         IsHexInputVisible="{TemplateBinding IsHexInputVisible}"
+                         MaxHue="{TemplateBinding MaxHue}"
+                         MaxSaturation="{TemplateBinding MaxSaturation}"
+                         MaxValue="{TemplateBinding MaxValue}"
+                         MinHue="{TemplateBinding MinHue}"
+                         MinSaturation="{TemplateBinding MinSaturation}"
+                         MinValue="{TemplateBinding MinValue}"
+                         PaletteColors="{TemplateBinding PaletteColors}"
+                         PaletteColumnCount="{TemplateBinding PaletteColumnCount}"
+                         Palette="{TemplateBinding Palette}"
+                         SelectedIndex="{Binding SelectedIndex, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                         ShowAccentColors="{TemplateBinding ShowAccentColors}" />
+            </Flyout>
+          </DropDownButton.Flyout>
+        </DropDownButton>
+      </ControlTemplate>
+    </Setter>
+  </Style>
+
+  <!-- Adjust Background within Flyout -->
+  <!-- Note: This is implemented but there seems to be an issue and the selector can't match across the Flyout -->
+  <Style Selector="ColorPicker /template/ ColorView#FlyoutColorView /template/ Border#TabBackgroundBorder">
+    <Setter Property="CornerRadius" Value="{DynamicResource TopOverlayCornerRadius}" />
+  </Style>
```

Comment:
Also note this doesn't work for some reason. I'm not sure if the comment is correct "This is implemented but there seems to be an issue and the selector can't match across the Flyout" or I made a mistake in the Selector (examples like this aren't in the docs but I did see some usage like this in other code).

Reply:
Selector by itself it valid. 

Similar limitation to this one https://docs.avaloniaui.net/docs/styling/troubleshooting#missing-style-pseudoclass-trigger-selector

Basically, TemplateBinding priority is higher than Style-Setter priority, but lower than Style-Trigger-Setter (with a pseudoclass)
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Data/BindingPriority.cs#L18-L37

Reply:
I would recommend leaving it as it is. It should be easier to work around it with control themes later.

Reply:
Yes, agree. I will have to rework the styles in control themes anyway so this will all be gone over again. 

Thanks for explaining how the priority is causing this situation.